### ISSUE TITLE
Skeleton/Bone support?

### ISSUE THREAD

**Issue by Salvakiya** [lawnjelly/godot-splerger#3]
does this already, or are there plans to, support bone/skeletons as well?

thinking about modular character systems where you swap out different parts of the character at runtime. you should be able to merge the correct meshes together and have the character use a single skeleton even if the character is made up of many different parts/meshes.

**Comment by lawnjelly** [comment]
No, there's no support for this in `splerger`, it doesn't handle bones / weights, it is actually very simple (core merging handles a lot more cases). If someone else wants to add this ability, it might not be a massive job. You'd probably have to make sure the skeleton RID and the bone IDs matched or were remapped. It's quite a good use case actually that wasn't in my mind at the time (though I've done this before in different engines in the past).

I'm not actually sure what would happen attempting to join two skinned meshes with 3.x core merging ( #61568 ) , I'm guessing it would fail, but it might be possible to get such a scenario working in a future modification. However there are as yet no plans for core merging in 4.x. clayjohn is keen for merging in 4 (especially for OpenGL), but reduz wants to wait and see if there is demand, because drawcalls are cheaper in vulkan, and the cost / benefits are less clearcut.
